posts/2021-04-14-async-vision-doc-shiny-future.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Brainstorming Async Rust's Shiny Future"
+author: Niko Matsakis
+description: "Brainstorming Async Rust's Shiny Future"
+team: the Async Foundations Working Group <https://rust-lang.github.io/wg-async-foundations/>
+---
+
+On March 18th, we [announced the start of the Async Vision Doc][announce] process. Since then, we've landed [24 "status quo" stories][sq] and we have [4 more stories in open PRs][prs]; [Ryan Levick] and [I] have also hosted more than ten collaborative writing sessions over the course of the last few weeks, and we have [more scheduled for this week][cws].
+
+[cws]: https://smallcultfollowing.com/babysteps/blog/2021/04/12/async-vision-doc-writing-sessions-v/
+
+Now that we have a good base of "status quo" stories, we are starting to imagine what the [✨ "shiny future" ✨][sf] might look like. **We want your help!** If you have a great idea for Async Rust[^youknow], then [take a look at the template and open a PR][template]! Alternatively, if you have an idea for a story but would like to discuss it before writing, you can [open a "shiny future" issue][open]. Also, we would still love to get more ["status quo" stories][sq], so please continue to share those.
+
+[^youknow]: Don't be modest. You know you do.
+
+When writing "shiny future" stories, the goal is to focus on the **experience** of Rust's users first and foremost, and not so much on the specific technical details. In fact, you don't even have to know exactly how the experience will be achieved. We have a few years to figure that out, after all. 🚀
+
+Every "shiny future" story is a "retelling" of one or more "status quo" stories. The idea is to replay the same scenario but hopefully with a happier ending, as a result of the improvements we've made. If you don't see a "status quo" story that is right for telling your "shiny future" story, though, that's no problem! Just write it now! We are still requesting "status quo" and "shiny future" stories, and we will do so right up until the end. 
```

Comment:
This seems fine as-is, but the "just write it now" here feels like it might discourage people from contributing shiny future stories if they don't want to go hunting for existing status quo stories or writing those. Is that intended? If not, maybe we can say something like "just make a note in your PR or issue..."?

Reply:
Yeah, I was looking back over the comments and saw your prior comment that was similar. I agree, I can tweak the language to make it more inviting. Will take a look later.